api/src/schemas.ts

```diff
@@ -43,6 +43,22 @@ export const schemas = {
       })
     }
   },
+  updateMyUsername: {
+    body: Type.Object({
+      username: Type.String({ minLength: 3, maxLength: 1000 })
+    }),
+    response: {
+      200: Type.Object({
+        message: Type.String(),
+        type: Type.Union([Type.Literal('success'), Type.Literal('info')]),
+        username: Type.Optional(Type.String())
```

Comment:
```suggestion
        username: Type.String()
```
It's always present on a success.